### ISSUE TITLE
home & garden update, start of year three

### ISSUE THREAD

**Issue by tjheffner** [tjheffner/heffdotdev#161]
---
title: "home & garden update"
date: 2022-05-27 00:00:00
category: DIY
tags: note garden sauna paint projects momentum
description: "year 3: the one about the yard"
---

Wow. The last two years have absolutely flown by. It's been a while since my last home updates ([1](/some-new-homeowner-tips) and [2](/first-time-homebuyer-recap))... _A Lot_ has been done in the interim. The coolest part for me is how each additional update or improvement seems to create even more momentum for the next project. the flywheel effect happening to my own motivation. I don't mind.

In two years, Randi and I have really turned this house into our home. I love living here with her so much. It truly is a personal paradise (aka "the purr palace" because our cat is a queen). And every little project we've done, big or small makes it that much better. 

## recap
to wit, over the last two years we've:
- painted every room but one (the bathroom, which eventually will be remodeled)
- built some initial [rain barrels](/rain-barrels)
- had the house re-piped
- built compost storage area, later expanded it a bit
- removed a ton of overgrown and overdone grasses left from previous owner. seriously 40 different puffballs, dotting the borders like little gumdrops. it was too much.
- upgraded to fiber & ran 5 drops through the crawlspace
- observed where the light and water go throughout the days and the seasons
- harvested many gallon freezer bags full of cherries, grapes, plums, raspberries, figs and more
- [set up the living room home theater](/home-theater-services)
- organized the garage with pegboards + better shelves
- learned how to replace the drain pump on a washing machine
- repaired a dishwasher & installed a new dishwasher several months later
- turned all of the front yard from grass into plantable space with the power of sheet mulching (thanks [chipdrop](http://chipdrop.net/)!)
- planted three trees (oak, hazelnut, nectarine) & four blueberry bushes, plus some grape cuttings in the newly plantable area
- bought a gas-powered generator after a [lengthy power outage](/lessons-from-the-storm)
- moved all the raspberries to one area of the yard
- added supports for the grapes + berries
- poured cement pad for hot tub
- ordered a sauna kit (12-16 weeks lead time)
- ripped out one existing raised bed
- cleaned up the street-strip with gravel box for trash cans + flowers instead of grass

Whew! That was a lot. And I am sure I am missing multiple somethings. We've been really busy! But it's been a good busy, and it's so wonderful to sit back and relax and savor the literal fruits of your labor. It's almost time to harvest cherries again :)

## what's next on the list? 
Obviously, lots of watering and weeding. Just a little bit of focused effort every day (at least 15 minutes, an hour is wonderful) really adds up in the end. It can be overwhelming to think about everything you have to do to get it where you want it, but [chunking it up](/one-box-at-a-time) has been a really effective technique for me. I like to focus on an area and then when it's done, see what my eye naturally wanders to next. That's my next project.
 
We are waiting for additional woodchips to finish sheet mulching a few areas of the yard (again courtesy of chipdrop, which provides them for free but not really on a set schedule of any sort). We've basically determined the raised beds that came with the house are nice, but not ideal. We need geriatric raised beds, I mean, really raise the things. The current ones are basically elevated by a foot, which is nice for separation, but still annoying to work in. We're going to pull most of them out and plant directly into the ground instead. The ones we don't remove, we're going to increase the height of. Maybe turn one into a [hugelkultur](https://www.almanac.com/what-hugelkultur-ultimate-raised-bed) box to help get the height up.

I also need to dig up a couple bushes in the back to prepare an area for the sauna's base. I want to have that all squared away before the kit is shipped. Summer goals. My plan is to create a small raised base, basically a small deck, so I can easily put a drain in the middle of the floor to hose it out in the future. It's going to be sited near the hot tub, mostly for electrical convenience.

One of these summer weekends I am going to build a [free little library](https://littlefreelibrary.org/), to go in the newly-cleaned up street area. I want to build it with a shelf below for extra vegetables from our garden, and maybe a space to share seeds as well.

I don't know if I'll get to it this year, but I want to upgrade the rain barrel situation, and figure out a more convenient way to use that water, preferably on a timer. Right now we have some drip lines that can be connected to a hose. I've also built a small pump-box from a plastic storage bin, a piece of 2x4, and a cheap pump from harbor freight. I connect it to the rain barrels and am able to water about half of the yard (particularly the berries, compost, and a fig and plum trees), but I'd like to cut myself out of the situation entirely, and have everything pumped on a timer from the rain barrels. That's gonna take a bit of research and some bigger barrels. I want to upgrade to 3 or 4 IBC containers on the side of the house. That would take my rainwater storage capacity from 110gal to 825-1100gal! and let me move the existing rain barrels elsewhere.

As you can see, the yard has been a big focus for us once we finished painting walls. So we haven't tackled any of the closet upgrades yet, but I hope to hit those in the coming months as well. Maybe towards the end of the year, when it isn't as pleasant to work outside... but it's real fun getting my hands dirty.

**Comment by ndouglas** [comment]
Wow! That's a crazy amount of work. This is the first I've heard of Hugelkultur, but I'm digging it. The generator: is that natural gas/whole house or gasoline?

**Comment by tjheffner** [comment]
Hugelkultur beds rock! I made one about a decade ago with a huge pile of crape myrtle branches, and that bed produced some of the absolute best tomatoes of my life. 

I'm not sure the specific model number but the generator is a portable tri-fuel generator from Firman that I got on sale at Costco. It says it can run on gasoline, natural gas or liquefied petroleum gas, but I'll probably stick to gasoline. It can hook into the main electrical panel and should be plenty powerful to run at least the furnace and fridge in the event our power goes out for extended periods again.
